eidas_node/tests/test_xml_pkcs11.py

```diff
@@ -0,0 +1,28 @@
+""" Do the same tests for signing and decrypting XML as in test_xml.py, but now with HSM setup and settings."""
+
+from unittest.mock import patch
+
+from eidas_node.tests import softhsm_setup, test_xml
+
+
+def setUpModule():
+    softhsm_setup.setup()
+
+
+def tearDownModule():
+    softhsm_setup.teardown()
+
+
+@patch.dict('eidas_node.tests.constants.SIGNATURE_OPTIONS',
+            {
+                'key_source': 'engine',
+                'key_location': 'pkcs11;pkcs11:token=test;object=test;pin-value=secret1',
+            })
+class TestSignXMLNodePKCS11(test_xml.TestSignXMLNode):
+    pass
```

Comment:
```suggestion
    """Tests with PKCS11 support."""
```

The `pass` statement sort of evokes skipped tests :)

Reply:
I added different (hope that better) docstring